exchanges/okcoin/okcoin_wrapper.go

```diff
@@ -29,46 +28,46 @@ import (
 )
 
 // GetDefaultConfig returns a default exchange config
-func (o *OKCoin) GetDefaultConfig(ctx context.Context) (*config.Exchange, error) {
+func (o *Okcoin) GetDefaultConfig(ctx context.Context) (*config.Exchange, error) {
 	o.SetDefaults()
 	exchCfg := new(config.Exchange)
 	exchCfg.Name = o.Name
 	exchCfg.HTTPTimeout = exchange.DefaultHTTPTimeout
 	exchCfg.BaseCurrencies = o.BaseCurrencies
 
-	err := o.SetupDefaults(exchCfg)
+	err := o.Setup(exchCfg)
 	if err != nil {
 		return nil, err
 	}
-
 	if o.Features.Supports.RESTCapabilities.AutoPairUpdates {
 		err = o.UpdateTradablePairs(ctx, true)
 		if err != nil {
 			return nil, err
 		}
 	}
-
 	return exchCfg, nil
 }
 
-// SetDefaults method assigns the default values for OKCoin
-func (o *OKCoin) SetDefaults() {
+// SetDefaults method assigns the default values for Okcoin
+func (o *Okcoin) SetDefaults() {
 	o.SetErrorDefaults()
-	o.Name = okCoinExchangeName
+	o.Name = okcoinExchangeName
 	o.Enabled = true
 	o.Verbose = true
 
 	o.API.CredentialsValidator.RequiresKey = true
 	o.API.CredentialsValidator.RequiresSecret = true
 	o.API.CredentialsValidator.RequiresClientID = true
-
 	requestFmt := &currency.PairFormat{Uppercase: true, Delimiter: currency.DashDelimiter}
 	configFmt := &currency.PairFormat{Uppercase: true, Delimiter: currency.DashDelimiter}
 	err := o.SetGlobalPairsManager(requestFmt, configFmt, asset.Spot, asset.Margin)
 	if err != nil {
 		log.Errorln(log.ExchangeSys, err)
 	}
-
+	err = o.DisableAssetWebsocketSupport(asset.Margin)
```

Comment:
Is margin actually supported? It actually doesn't seem like it. The documentation constantly has the following:
`Not enabled. Please disregard.`

Endpoints such as `https://www.okcoin.com/api/v5/public/instruments?instType=MARGIN` return no data, your ticker & orderbook data is sourced from SPOT data and there is no margin type for websocket

Unfortunately, I am unable to log in to okcoin on any of my accounts/new account to fully verify, but I'm leaning towards its not supported.

Reply:
I added that because the exchange uses margin(trade) modes like "cash", "isolated", "cross"

Reply:
I'll try and get access to okcoin today to see if it has any margin abilities

Reply:
<img width="566" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9261323/7600bbed-e2d4-4da8-b53c-60fb3c09573c">
Confirmed in writing. Thanks for already removing support!